sort/tsh_sort_1_0.py

```diff
@@ -0,0 +1,64 @@
+#Some kind of a sort
+#The sort checks accounts for missing numbers
+#and last number in array being smaller
+#
+# Array to be sorted
+a = [10,2,6,7,3,5,8,1,13,15,4]
+#
+#Initalize variables
+minIndex = 0
+maxIndex = len(a)
+lowestNum = min(a)
+highestNum = max(a)
+innerIndex = 0
+outerIndex = 0
+innerBuff = 0
+buff = 0
+n = 0
+#
+#Print initial Conditions
+print "Inner Loop index is ", innerIndex
+print "Outer Loop index is ", outerIndex
+print "minIndex is         ", minIndex
+print "maxindex is         ", maxIndex
+print "lowestNumber is     ", lowestNum
+print "highestNumber is    ", highestNum
+print "Temp buffer is      ", buff 
+#
+print "###################################"
+print ""
+#
+#start of sort
+print "Array to be sorted"
+print a
+print ""
+#Outer Loop search through array
+#Finds the number to be ordered
+while outerIndex < maxIndex:
+#

```

Comment:
The second you enter a while block, the line that preceeds it should be indented.  Consider getting a python linter, it will actually fix this automagically for you on save.